### PR TITLE
Html project

### PR DESCRIPTION
**Pull request by onyedikamike**
My Landing_page.

- I build a landing page using the necessary html and css element.
- Though it was challenging but i understood that you can only get the right result through using the right commands.
- Using the flexbox properties, makes the project became interesting and awesome.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        ><img class="img_1" src="images/food logo.jpg" alt="food logo"
+      /></a>
+      <nav>
+        <ul class="link_container">
+          <li><a href="#">Home</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
```
Comment (bobb-Rob) [inline index.html:22]: - [x] Please apply the inter-font-family to these link items.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+          African foods are bountiful and diverse. They are rich in nutritional
+          fiber and often unrefined and they offer a healthy choice when
+          consumed in the right blend.
+        </p>
+        <button class="btn_signup" type="button">sign up</button>
+      </div>
+      <div>
+        <img src="images/catering.jpg" alt="African foods" />
+      </div>
+    </section>
```
Comment (bobb-Rob) [inline index.html]: Your class name for this section is not descriptive enough. Using `sect` does not give us an idea of what this section is all about. This is essential for you and others that will work with you on future projects. 
- [x] Please give this section a descriptive class-name. 🤝

Comment (bobb-Rob) [inline index.html]: - [x] The height of this section on your design as shown below 👇 is much smaller than that of the template. Please look into this and fix it. One way you can fix this using margins,..Do not give it an explicit height, use margins instead.

image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+          <li>
+            <div class="image_contains">
+              <img src="images/caterers4.jpg" alt="Fourth caterer" />
+              <h3>Peter Elvis</h3>
+              <p>Abuja, expert in edikanikon</p>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </section>
```
Comment (bobb-Rob) [inline index.html:71]: - [x] The images in this section are way too small, you can look at the template and try to make it look alike. This is an essential skill to be able to replicate product requirements for your clients.  Follow the video I did, see how I used images that are large enough,...

image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    <section class="section_3">
+      <p>
+        Traditionally most of Africa dishes use a combination of locally cereal
+        grains, available fruits and vegetables, as well as meat and milk
+        products. In some parts of the Africa, the traditional diet features a
+        preponderance of curd, fresh and healthy vegetables.
+      </p>
```
Comment (bobb-Rob) [inline index.html:78]: - [x] This section text does not start from the left and right edges, you could use width percentage. e.g 60% width.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @onyedikamike , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html (hunk 4 above)
Comment (onyedikamike) [reply index.html:78]: All corrections have been attended to sir. Thanks for the observations

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @onyedikamike , 

### you have earned 4pts....Good job
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+            <div class="image_contains">
+              <img src="images/caterers3.jpg" alt="Third caterer" />
+              <h3>Nkechi Dan</h3>
+              <p>Rivers, awesome cook</p>
+            </div>
+          </li>
+          <li>
+            <div class="image_contains">
+              <img src="images/caterers4.jpg" alt="Fourth caterer" />
+              <h3>Peter Elvis</h3>
```
Comment (onyedikamike) [inline index.html:65]: Kindly disregard

**Review by onyedikamike — COMMENTED** [review]
Approved

**Pull request merged**